source/draw/gpu/opengl/backend/gles3/Gles3Context.ooc

```diff
@@ -20,27 +20,29 @@ Gles3Context: class extends GLContext {
 	_contextCount := static 0
 	_mutex := static Mutex new()
 	init: func { super() }
+	validate: static func (value, expectedValue: UInt, function: String) {
+		if (value != expectedValue)
+			Debug error(function + " failed! Expected status %u but got %u. eglError=%d" format(expectedValue, value, eglGetError()))
+	}
+	validate: static func ~expression (success: Bool, function: String) {
+		if (!success)
+			Debug error(function + " failed with eglError=%d" format(eglGetError()))
```

Comment:
Sure, but we're throwing an exception right after (since this is `Debug error`), so doesn't really matter.